places/Portadown.md

```diff
@@ -0,0 +1,5 @@
+### Portadown 
+
+- County Armagh
+- Around 30,000 people
```

Comment:
Not the last few days because Lurgan people putting suspicious devices everywhere @MichaelReel